requirements.txt

```diff
@@ -16,6 +16,7 @@ numpy
 platformdirs>=2.3.0
 psutil
 pydantic>=2
+pylint
```

Comment:
This should go in the `[dev]` group in pyproject.toml rather than in the main `requirements.txt`

Reply:
Done